django_celery_tracker/views.py

```diff
@@ -0,0 +1,104 @@
+import json
+from datetime import timedelta
+from django.core.serializers.json import DjangoJSONEncoder
+from django.db.models import Q
+from django.shortcuts import get_object_or_404
+from django.template.response import HttpResponse, SimpleTemplateResponse
+from django.utils.dateparse import parse_datetime
+from celery.result import AsyncResult
+
+from django_celery_tracker.decorators import admin_required
+from django_celery_tracker.helpers import (
+    get_task_created_item,
+    get_task_data,
+)
+from django_celery_tracker.models import CeleryTask
+
+
+@admin_required
+def tracker_dashboard(request):
+    context = {
+        "title": "Celery Dashboard",
+        "has_permission": True,
+        "user": request.user,
+        "test": "asdf",
```

Comment:
Don't see this value being used anywhere, is it still needed?

Reply:
Good call.